neard/src/runtime.rs

```diff
@@ -451,6 +451,7 @@ impl NightshadeRuntime {
             cache: Some(Arc::new(StoreCompiledContractCache { store: self.store.clone() })),
             #[cfg(feature = "protocol_feature_evm")]
             evm_chain_id: self.evm_chain_id(),
+            profile: None
```

Comment:
is it better if this is hidden behind a feature flag just like `evm_chain_id`?

Reply:
Having discussed this with @olonho, he made a good point that it might be worth having in general as it could be used in a more detailed outcome via RPC.  I had initially hidden it behind a flag and could switch back.

Reply:
I agree it's better not to hide it. protocol_feature_xxx is used when we eventually want to add this feature to nodes but not right now. This is a feature that is not a protocol upgrade and is a feature we want to enable right now

Reply:
It is not related to the protocol features. It is more about the fact that we don't need `profile` for the node to function in its normal setting. Is the current approach using a runtime flag to guard this?

Reply:
There is already a `costs_counting` feature that will prevent the node from keeping track of the fees in `VMLogic`.  So the "tax" on the nodes is passing around a None value. Currently the main `run` method in the runner checks if the value is `Some` and `run_vm_profiled`.

Would you prefer that this join the `cost_counting` feature and only check for option type with it?

Reply:
Okay then I think it makes sense to put this behind `cost_counting` as well.